File: Swift/Leetcode.playground/Sources/2_String/LC_2678_NumberOfSeniorCitizens.swift
class Solution {
    func countSeniors(_ details: [String]) -> Int {
        var count = 0
        for info in details {
            let index1 = info.index(info.startIndex, offsetBy: 11) 
            let index2 = info.index(info.startIndex, offsetBy: 12) 
            if let age = Int(info[index1...index2]), age > 60 {
                count += 1
            }
        }
        return count
    }
}

File: Java/2_String/LC_2678_NumberOfSeniorCitizens.java
class Solution {
    public int countSeniors(String[] details) {
        int count = 0;
        for (String detail : details) {
            int age = Integer.parseInt(detail.substring(11, 13));
            if (age > 60) {
                count++;
            }
        }
        return count;
    }
}

public class LC_2678_NumberOfSeniorCitizens {

}
